src/index.html

```diff
@@ -3,14 +3,27 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="//db.onlinewebfonts.com/c/c862d8098ce50a7dc042ebbb9b94bbc8?family=TwCenW01-Medium" rel="stylesheet" type="text/css"/> 
     <link rel="stylesheet" href="css/style.scss">
-    <title>Document</title>
+    <link rel="stylesheet" media="screen" href="https://fontlibrary.org/face/germano" type="text/css"/> 
+    <title>places-been</title>
 </head>
 <body>
+        <nav class='navbar '>
+            <h4 class="navbar__title">places been</h4>
+            <h4 class='navbar__subtitle'>a travel journal</h4>
+        </nav>
+        <div class="buttons container">
+            <h4 class=buttons__view>View:</h4>
+            <button class='button visited-button' type='button'>Visited</button>
+            <button class='button bucketlist-button' type='button'>Bucketlist</button>
+            <button class='button all' type='button'>Show all</button>
+        </div>
```

Comment:
A little about A11Y (Accessibility):

Headings should follow a semantic structure, starting from h1 up to h6 and should be used in a meaningful way. Don't use a heading just because it "looks like" a heading. My general tip to find the best heading for content is to imagine the page without styles, as if it was a book. What heading would you use? This is something that comes with experience though.

Here's a short [google article about headings and landmarkst](https://web.dev/headings-and-landmarks/).

When you are done reviewing the structure of this page, let me know so that I can give further feedback ;)

Reply:
Done. I had doubts whether to leave ```nav``` or replace it with ```header``` but I suppose it will have a navigation functionality at one point

Reply:
Any additional feedback here?

Reply:
The `nav` should indeed be a `header` instead. It's not a nav, because it does not contain anything related with navigation ATM. In the future, we can create a `nav` inside the `header`.

Reply:
Got it. If that's all, do you think we can move on with the next part of this project? I'm all out of ideas 😄

Reply:
Yes, please check my latest comment (was posted a few minutes later) saying exactly that :D